woocommerce/woocommerce-ios
# [Mobile Payments] Remove stripe IPP from experiments; always true now

Closes: #6204

@koke @joshheald - just one review needed but either of y'all are welcome

fyi @malinajirka 

### Description
- Removes Stripe IPP from experiments
- Removes feature flag for Stripe IPP
- `rake generate` was required to update copiables
- Updates `CardPresentPaymentsConfiguration` to just treat it always as enabled/true
- Carefully merges the related changes on trunk for https://github.com/woocommerce/woocommerce-ios/pull/6195

### Testing instructions
- Ensure unit tests pass
- Ensure you can access IPP flows on a store powered by the Stripe extension

### Screenshots
- Experiments looks like this now:

![IMG_0314](https://user-images.githubusercontent.com/1595739/154569052-d9d49ace-ef85-4b1b-b901-caa596c1bf99.PNG)

---
